pkg/controller/plan/migration.go

```diff
@@ -1347,6 +1348,8 @@ func (r *Predicate) Evaluate(flag libitr.Flag) (allowed bool, err error) {
 		_, allowed = r.vm.FindHook(PreHook)
 	case HasPostHook:
 		_, allowed = r.vm.FindHook(PostHook)
+	case RequiresConversion:
+		allowed = r.context.Source.Provider.Type() == api.VSphere
```

Comment:
once/if we add more sources that virt-v2v supports (xen, hyperv) then it will be nice to have that closer to the definition of the provider type, but for the time being that's ok